plugins/pulp_rpm/plugins/importers/yum/listener.py

```diff
@@ -196,6 +196,7 @@ def download_succeeded(self, report):
                 return
             added_unit = self.sync.add_rpm_unit(self.metadata_files, unit)
```

Comment:
Yup, I definitely wasn't angling that we try to fix either issue as part of this work :shipit: